tests/integration/test_charm_basic.py

```diff
@@ -0,0 +1,198 @@
+# Copyright 2022 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""Tests for basic Jenkins functionality."""
+
+import secrets
+
+from ops.model import ActiveStatus, Application
+import pytest
+
+from pytest_operator.plugin import OpsTest
+import jenkins
+
+from .types import JenkinsCredentials
+from .config import PLUGINS_DIR
+
+pytestmark = pytest.mark.basic
+
+
+@pytest.mark.asyncio
+@pytest.mark.abort_on_fail
+async def test_active(app: Application):
+    """
+    arrange: given charm has been deployed and is idle
+    act: when the status is checked
+    assert: then the status is active and a message which contains running has been set
+    """
+    assert app.units[0].workload_status == ActiveStatus.name
+    assert "running" in app.units[0].workload_status_message
+
+
+@pytest.mark.asyncio
+@pytest.mark.abort_on_fail
+async def test_debian_stable_in_sources(app: Application):
+    """
+    arrange: given charm has been deployed and is idle
+    act: when /etc/apt/sources.list is checked
+    assert: then it contains debian-stable
+    """
+    souces_list_contents = await app.units[0].ssh("cat /etc/apt/sources.list")
+
+    assert "debian-stable" in souces_list_contents, "LTS not in sources.list"
+
+
+@pytest.mark.asyncio
+@pytest.mark.abort_on_fail
+async def test_tools_installed(app: Application):
+    """
+    arrange: given charm has been deployed and is idle
+    act: when checking for the python3-testtools package
```

Comment:
I'm not sure I understand why we're testing for this particular package. Could we explain that somewhere here?

Reply:
The next commit should add a clarifying note based on a guess (that it is a required tool), this was taken straight from the previous amulet based tests which unfortunately didn't include an explanation. Included here so the integration tests don't regress